Mrj final review ch5 and 7 (#399)
worldbank/d4di: chapters/data-collection.tex

\begin{fullwidth}
Much of the recent push toward credibility in the social sciences has focused on analytical practices.
However, credible development research often depends, first and foremost, on the quality of the raw data.
When you are using original data - whether collected for the first time through surveys or sensors or acquired through a unique partnership -

> there is no way for others to validate that it accurately reflects reality

Reviewer: There is something about this sentence that rubs me a little the wrong way. I think it was fine to say "no way for others" when we talked about data collection only, but but when we include admin data it does not feel as right, as the civil servants that gave us the data could validate it. How about this suggestion:

```suggestion
there is no way for anyone else in the research community to validate that it accurately reflects reality
```

and that the indicators you have based your analysis on are meaningful.
This chapter details the necessary components for a high-quality data acquisition process,
no matter whether you are receiving large amounts of unique data from partners
or fielding a small, specialized custom survey. By following these guidelines, you will be able to move on to data analysis,
assured that your data has been obtained at high standards of both quality and security.

The chapter begins with a discussion of some key ethical and legal descriptions
to ensure that you have the right to do research using a specific dataset.

> Particularly when confidential data is being collected at your behest

Reviewer: **behest**

```suggestion
Particularly when confidential data is collected by you and your team
```

Author: behest really is the right word here ...

and to fully inform others of what you are doing.
Writing down and agreeing to specific details is a good way of doing that.

If the research team requires access to existing data for novel research,
terms of use should be agreed on with the data owner, 
typically through a data licensing agreement.
Keep in mind that the data owner is likely not familiar
with the research process, and therefore may be surprised
at some of the things you want to do if you are not clear up front.

> You will typically want intellectual property rights to all derivative works developed used the data,

Reviewer: I am not sure what the extent of what we are saying here is. What is included in "derivative works"? Are those research outputs, or are they also data sets generated during cleaning/construct? 

Can we use a easier and more direct term here? What about?

```suggestion
You will typically want intellectual property rights to all research outputs developed used the data,
```

a license for all uses of derivative works, including public distribution 
(unless ethical considerations contraindicate this).
This is important to allow the research team to store, catalog, and publish, in whole or in part,

> either the original licensed dataset or the derived materials.

Reviewer: I think it is much more helpful for a reader new to this topic if we are more precise in the examples we are listing.

```suggestion
either the original licensed dataset or just the newly constructed variables.
```

Author: changed to "or datasets derived from the original."

> either the original licensed dataset or the derived materials.

Author: ```suggestion
either the original licensed dataset or datasets derived from the original.
```

Make sure that the license you obtain from the data owner allows these uses,
and that you consult with the owner if you foresee exceptions with specific portions of the data.

If the research team contracts a vendor to collect original data,
contract terms must clearly stipulate that the research team owns the data
and maintains full intellectual property rights. 
The contract should also explicitly stipulate that the contracted firm
is responsible for protecting the confidentiality of the respondents,

> and that the data collection will not be distributed to any third parties

Reviewer: I had to read twice to understand that it was the practical task of collecting the data that we are talking about here.

```suggestion
and that the data collection will not be delegated to any third parties
```

or used by the firm or subcontractors for any purpose not expressly stated in the contract,
before, during or after the assignment. 
The contract should also stipulate that the vendor is required to comply with 
ethical standards for social science research, 
and adhere to the specific terms of agreement with the relevant 
Institutional Review Board or applicable local authority. 

Research teams that collect their own data must consider the terms
under which they will release that data to other researchers or to the general public. 

> Will you publicly releasing the data in full (removing personal identifiers))?

Reviewer: ```suggestion
Will you publicly releasing the data in full (removing personal identifiers)?
```

Would you be okay with it being stored on servers anywhere in the world,

> even ones that are owned by corporations or governments abroad?

Reviewer: What is abroad in this sentence? The US is a [5-eyes](https://restoreprivacy.com/5-eyes-9-eyes-14-eyes/) nation, so it would be weird if it comes across as we are saying "abroad" from an American perspective.

```suggestion
even ones that are owned by corporations or governments in countries other than your own?
```

> Would you prefer to decided permission on a case-by-case basis, dependent on specific proposed uses? 

Reviewer: ```suggestion
Would you prefer to decide permission on a case-by-case basis, dependent on specific proposed uses? 
```

and it is not possible to keep track of these kinds of information as data over time.
Eventually, you will want to make sure that you are creating a collection or object
that can be properly submitted to a data catalog and given a reference and citation.
The metadata - documentation about the data - is critical for future use of the data.
Metadata should include documentation of how the data was created, 
what they measure, and how they are to be used.
In the case of survey data, this includes the survey instrument and associated manuals; 
the sampling protocols and field adherence to those protocols, and any sampling weights;
what variable(s) uniquely identify the dataset(s), and how different datasets can be linked;
and a description of field procedures and quality controls. 
We use as a standard the Data Documentation Initiative (DDI), which is supported by the 

> World Bank's Microdata Catalog.\sidenote{\url{microdata.worldbank.org}}

Reviewer: ```suggestion
World Bank's Microdata Catalog.\sidenote{\url{https://microdata.worldbank.org}}
```
